apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/soeknadoversikt/virkningstidspunkt/Virkningstidspunkt.tsx

```diff
@@ -50,6 +50,8 @@ const Virkningstidspunkt = (props: Props) => {
     props.virkningstidspunkt ? new Date(props.virkningstidspunkt.dato) : null
   )
   const [virkningstidspunkt, fastsettVirkningstidspunktRequest] = useApiCall(fastsettVirkningstidspunkt)
+  const [begrunnelse, setBegrunnelse] = useState<string>(props.virkningstidspunkt?.begrunnelse || '')
```

Comment:
Veeeldig minor, men til info så bør man bruke `??` i optional chaining, i steden for `||`. Førstnevnte er final state i optional chaining, men sistnevnte bare sjekker om svaret er falsy.
```
false || true => true
false ?? true => false
```